Extensions/Today/Images.xcassets/subtitleLableColor.colorset/Contents.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
Can get rid of colorset since you are using TodayUX.labelColor and subtitleLabelColor